grpc/src/main/java/com/linecorp/armeria/internal/grpc/HttpStreamReader.java

```diff
@@ -95,6 +98,22 @@ public void onNext(HttpObject obj) {
             // Only clients will see headers from a stream. It doesn't hurt to share this logic between server
             // and client though as everything else is identical.
             HttpHeaders headers = (HttpHeaders) obj;
+
+            if (!sawLeadingHeaders) {
+                sawLeadingHeaders = true;
+                if (headers.status() == null) {
```

Comment:
Shouldn't we do something similar to what we did in: https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/common/HttpResponseAggregator.java#L54 ?

Reply:
Thanks - added skipping of informationals and made error message for missing status code same as upstream